plenum/config.py

```diff
@@ -44,6 +44,7 @@
 seqNoDbName = 'seq_no_db'
 
 nodeStatusDbName = 'node_status_db'
+instanceChangeDbName = 'instance_change_db'
```

Comment:
Do we really need a separate DB here? I think we can use `node_status_db` to avoid too many rocksdb databases